### PR TITLE
Adding Lessons to content issue templates

### PR DESCRIPTION
**Pull request by jonathanbossenger**
Along with adding #2120, this updates the issue templates to include lessons.

### CODE REVIEW

**Comment by jonathanbossenger** [comment]
Note that we can only merge this once the new [Lesson Creation guidelines](https://docs.google.com/document/d/1twvDGmVZsyjBZbi9abiqoA_a2G7O_WNis_iGL2Sccg0/edit?usp=sharing) have been published.

**Review by krupal-panchal — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/01-content-development-general.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ A team member will review the proposal and share next steps.
 - Topic description: 
 - Audience (User, Developer, Designer, Contributor, etc.): 
 - Learning objectives (What will the learner be able to do as a result of this content?): 
-- Content type (Tutorial, Online Workshop, Course or Lesson Plan): 
+- Content type (Course, Lesson, Tutorial, Online Workshop, or Lesson Plan):
```
Comment (kaitohm) [inline .github/ISSUE_TEMPLATE/01-content-development-general.md]: Minor detail, but the previous order of these content types reflected the amount of work needed to create them, and the priority in the Training Team. Psychologically, people are prone to choose the early options if they're not sure what to choose.

Based on that thought, what about reordering these as follows?
```suggestion
- Content type (Online Workshop, Lesson, Course, Tutorial, or Lesson Plan): 
```

File: .github/ISSUE_TEMPLATE/02-content-development-faculty.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ labels: Awaiting Triage
 
 # Details
 
-- Content type (Course, Tutorial, Online Workshop, or Lesson Plan):
+- Content type (Course, Lesson, Tutorial, Online Workshop, or Lesson Plan):
```
Comment (kaitohm) [inline .github/ISSUE_TEMPLATE/02-content-development-faculty.md]: Reordering to match the general template.

```suggestion
- Content type (Online Workshop, Lesson, Course, Tutorial, or Lesson Plan):
```

File: .github/ISSUE_TEMPLATE/02-content-development-faculty.md [hunk 3]
```diff
@@ -23,5 +23,5 @@ What will the learner be able to do as a result of this content?
 
 ### Automation Code
 <!-- 
-Under this comment, type two slashes (//) followed by the content type (tutorial, online-workshop, lesson-plan, course). (Like //example) Once submitted, that code will add a new comment to the issue with the relevant development checklist.
+Under this comment, type two slashes (//) followed by the content type (lesson, tutorial, online-workshop, lesson-plan, course). (Like //example) Once submitted, that code will add a new comment to the issue with the relevant development checklist.
```
Comment (kaitohm) [inline .github/ISSUE_TEMPLATE/02-content-development-faculty.md]: Reordering to match the general template. Also, updating code for Lesson Plans.

```suggestion
Under this comment, type two slashes (//) followed by the code that represents the content type. (Like //example) 
- Online Workshop = online-workshop
- Lesson = lesson
- Course = course
- Tutorial = tutorial
- Lesson Plan = teach
Once submitted, that code will add a new comment to the issue with the relevant development checklist.
```

**Review by kaitohm — APPROVED** [review]
I've modified the order of the listed content, and also updated the command for lesson plans.

File: .github/ISSUE_TEMPLATE/01-content-development-general.md (hunk 1 above)
Comment (jonathanbossenger) [reply .github/ISSUE_TEMPLATE/01-content-development-general.md]: Ah, thanks for clarifying, I was not aware there was a specifically predetermined order.

**Comment by jonathanbossenger** [comment]
@bsanevans checking in here, is this ready to be merged?

**Pull request merged**